pkg/apis/networking/types.go

```diff
@@ -55,6 +55,16 @@ type NetworkPolicySpec struct {
 	// solely to ensure that the pods it selects are isolated by default)
 	// +optional
 	Ingress []NetworkPolicyIngressRule
+
+	// List of egress rules to be applied to the selected pods. Outgoing traffic is
+	//allowed unbounded if there are no NetworkPolicies selecting the pod
+	// (and cluster policy otherwise allows the traffic), OR if the traffic source is
+	// the pod's local node, OR if the traffic matches at least one egress rule
+	// across all of the NetworkPolicy objects whose podSelector matches the pod. If
+	// this field is empty then this NetworkPolicy does not limit any outgoing traffic
+	// (and serves solely to ensure that the pods it selects are isolated by default)
+	// +optional
+	Egress []NetworkPolicyEgressRule
```

Comment:
Do we have a comment indicating 'beta' or 'alpha' ?  @lavalamp since machinery team seems to want people to just use fields rather than annotations, shouldn't the generated docs at least say "beta" ?

Reply:
Didn't we decide we need a field to discriminate whether this policy was applying ingress vs egress?

Reply:
Yes @danwinship mentioned that above. Adding now.

Reply:
Yes it should be beta, but I was not sure if the format was supposed to be similar to the feature gate note or not. 

```
// owner: @cmluciano
// beta: v1.8
```

Reply:
I've noticed this doesn't mention local node. What's the behavior for `if the traffic destination is the pod's local node`?